x-pack/plugins/enterprise_search/public/applications/enterprise_search_content/components/search_index/pipelines/ml_inference/model_select_option.tsx

```diff
@@ -49,7 +51,7 @@ export const MlModelSelectOption: React.FC<MlModelSelectOptionProps> = ({ model
             <EuiFlexGroup gutterSize="xs">
               <EuiFlexItem>
                 <span>
-                  <EuiBadge color="hollow">{type}</EuiBadge>
+                  <EuiBadge color={isTextExpansionModel(model) ? "success" : "hollow"}>{typeFriendlyName}</EuiBadge>
```

Comment:
Any (simple) suggestions on vertically centering the badge to be in line with the model type and the status would be appreciated 🙂 
![Screenshot 2023-03-28 at 3 01 30 PM](https://user-images.githubusercontent.com/14224983/228351447-d9f19fe9-291d-4b8e-929b-70e34056e0aa.png)